src/System.IO.FileSystem/tests/Directory/CreateDirectory.cs

```diff
@@ -127,7 +127,7 @@ public void ValidPathWithTrailingSlash()
         }
 
         [ConditionalFact(nameof(UsingNewNormalization))]
-        [SkipOnTargetFramework(TargetFrameworkMonikers.Uap | TargetFrameworkMonikers.UapAot)]
+        [SkipOnTargetFramework(TargetFrameworkMonikers.Uap)]
```

Comment:
This was due to a lack of long path support in the WinRT APIs (some were limited by MAX_PATH). I need to create an issue to track this for 2.1 as API support should be in place and we'll be removing a chunk of WinRT APIs from our end.